### DISCUSSION TITLE
https://doesmycode.work/posts/using-onedev-for-repository-backups/

### DISCUSSION THREAD

**Discussion by giscus** [steveiliop56/steveiliop56.github.io#1] [Comments]
# https://doesmycode.work/posts/using-onedev-for-repository-backups/

You probably have seen the recent GitHub Outage were GitHub was down globally! I was never afraid that I would lose any data but I thought “What if something happened and some of my repositories were affected?”. I don’t have any backups so the thought of losing the code I spent hours or days working on and that I use as reference to my new projects was a bit scary. So, let’s make offline mirrors!

https://doesmycode.work/posts/using-onedev-for-repository-backups/

**Comment by steveiliop56** [comment]
Oh wow it really works
